1001-2000/LIT1295Dersan.xml

```diff
@@ -138,413 +146,1482 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                        ኵሉ፡ አውራኅ፡ ትንብልናሁ፡ የሀሉ፡ ምስሌነ፡ ለዓለመ፡ ዓለም፡ አሜን።</l>
+                  </ab>
                </div>
-               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="SalamYakkatit">
-                  <label xml:lang="en">
-                     <foreign xml:lang="gez">Salām</foreign> for <foreign xml:lang="gez">Yakkātit</foreign>
-                  </label>
+               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="SalamBeauties">
+                  <label xml:lang="en">I say salām to your ordered beauties</label>
+                  <!-- As witnessed in <ref type="mss" corresp="BAVet82"/>, but wanting in <ref type="mss" corresp="BMLacq776"/>.-->
+                  <ab>
+                     <l n="60">፷፩፤ ሰላም፡ እብል፡ ሠናይተከ፡ ተአዛዜ፡ ለትፍሥሕተ፡ ሰብእ፡ ወለኑዛዜ። ሚካኤል፡ ኢይበል፡ አድኅኖተከ፡ ማእዜ። ረሲ፡ ደኃሪትየ፡ እምግባረ፡ ኀጢአት፡ አግዓዜ።
+                        እንተ፡ ተቀነይኩ፡ ሎቱ፡ የአክል፡ እምይእዜ።</l> 
+                  </ab>
                </div>
             </div>
             
             <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="Maggabit">
-               <label xml:lang="en">
-                  <foreign xml:lang="gez">Maggābit</foreign>
-               </label>
-               
-               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="HomilyMaggabit">
-                  <label xml:lang="en">Homily for <foreign xml:lang="gez">Maggābit</foreign> by a patriarch of Antioch</label>
-                  <listBibl type="editions">
-                     <bibl corresp="#BLorient691">
-                        <ptr target="bm:budge1894stmichael"/>
-                        <citedRange unit="page">195-216</citedRange>
-                     </bibl>
-                  </listBibl>
+               <label xml:lang="en">Maggābit</label>
+               <div type="textpart" subtype="part" xml:id="AdmonitionMaggabit">
+                  <label>Admonition for Maggābit</label>
+                  <ab>በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ አሐዱ፡ አምላክ፡ ድርሳን፡ ወነገር፡ ዘቅዱስ፡ ወክቡር፡ ወኀያል፡ መልአክ፡ ሚካኤል፡ ሊቀ፡ መላእክት፡ ዘይትነበብ፡ አመ፡ ዐሡሩ፡ ወሰኑዩ፡
+                     ለወርኀ፡ መጋቢት፡ ዘደረሰ፡ አባ፡ ዮሐንስ፡ ሊቀ፡ ጳጳሳት፡ ዘአንጾኪያ፡ በእንተ፡ ዕበዩ፡ ወክብሩ፡ ለሊቀ፡ መላእክት፡ ሚካኤል፡ ወበእንተ፡ አፍቅሮቱ፡ ሰብአ፡ ወበእንተ፡ ሰንበት፡ ቅድስት፡
+                     ተናገረ፡ እስመ፡ በዓለ፡ ሚካኤል፡ ኀብረ፡ በይእቲ፡ ዓመት፡ ምስለ፡ ዕለተ፡ ሰንበት፡ ወካዕበ፡ ተናገረ፡ በእንተ፡ ማቴዎስ፡ ነግድ፡ ወብእሲቱ፡ ወውሉዱ፡ ዘከመ፡ አምኑ፡ በእግዚአብሔር፡
+                     በስእለቱ፡ ለቅዱስ፡ ሚካኤል፡ ሊቀ፡ መላእክት፡ እስመ፡ ይቤ፡ ዝንቱ፡ ድርሳን፡ አመ፡ ዐሡሩ፡ ወሰኑዩ፡ ለወርኀ፡ መጋቢት፡ እንዘ፡ ጉቡኣን፡ ሕዝብ፡ በቤተ፡ ክርስቲያኑ፡ ለቅዱስ፡ ሚካኤል፡
+                     ሊቀ፡ መላእክት፡ ከመ፡ ክርስቲያን፡ ይግበሩ፡ በዓሎ፡ በሰላመ፡ እግዚአብሔር፡ አብ፡ ትንብልናሁ፡ ወአስተብቍዖቱ፡ የሀሉ፡ ምስሌነ፡ ለዓለመ፡ ዓለም፡ አሜን።</ab>
+               </div>
+               <div type="textpart" subtype="chapter" xml:lang="gez" xml:id="HomilyPsalm">
+                  <label xml:lang="en">Homily by a patriarch of Antioch on the Psalm of David</label>
+                  <note>The homily was edited by <bibl corresp="#BLorient691"><ptr target="bm:budge1894stmichael"/>
```

Comment:
remove `#`, 
`#` is only for `target`, a pointer to an anchor (sub-id) inside of the same file you are in